src/dto/Network.ts

```diff
@@ -15,7 +15,7 @@ export enum Network {
   CRONOS = 'cro-mainnet',
   DOGECOIN = 'doge-mainnet',
   EOS = 'eos-mainnet',
-  EON = 'eon-mainnet',
+  HORIZEN_EON = 'horizen-eon-mainnet',
```

Comment:
This will break `address` module for Eon as core-api data api is expecting value 'eon-mainnet'